test/powershell/Modules/Microsoft.PowerShell.Utility/Read-Host.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+if ( ! (get-module -ea silentlycontinue TestHostCS ))
```

Comment:
There are existing Read-Host tests in `Host/Read-Host.Tests.ps1`; do you want to consolidate them?